docs/source/index.md

```diff
@@ -44,13 +44,17 @@ You are also welcome to join discussions about using PeekingDuck in the followin
 
 AI models are cool and fun, but we're even more interested to use them to solve real-world problems. We've combined dabble nodes with model nodes to create **use cases**, such as [social distancing](https://aisingapore.org/2020/06/hp-social-distancing/) and [group size checking](https://aisingapore.org/2021/05/covid-19-stay-vigilant-with-group-size-checker/) to help combat COVID-19. For more details, click on the heading of each use case below.
 
-| | |
-|-|-|
-| [Social Distancing](use_cases/social_distancing.md) | [Zone Counting](use_cases/zone_counting.md) |
-|<img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/social_distancing.gif" width="100%"> |<img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/zone_counting.gif" width="100%">|
-| [Group Size Checking](use_cases/group_size_checking.md) | [Object Counting](use_cases/object_counting.md) |
-|<img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/group_size_check_2.gif" width="100%">|<img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/object_counting.gif" width="100%"> |
-| | |
+|                                                              |                                                              |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Social Distancing](use_cases/social_distancing.md)          | [Zone Counting](use_cases/zone_counting.md)                  |
+| <img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/social_distancing.gif" width="100%"> | <img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/zone_counting.gif" width="100%"> |
+| [Group Size Checking](use_cases/group_size_checking.md)      | [Object Counting](use_cases/object_counting.md)              |
+| <img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/group_size_check_2.gif" width="100%"> | <img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/object_counting.gif" width="100%"> |
+| [Privacy Protection Faces](use_cases/privacy_protection_faces.md) | [Privacy Protection License Plates](use_cases/privacy_license_plate.md) |
+|                                                              |                                                              |
+| [Face Mask Detection](use_cases/face_mask_detection.md)      |                                                              |
+|                                                              |                                                              |
+|                                                              |                                                              |
```

Comment:
This table needs to be added to README.md as well

Reply:
Added the links to the mask detection gif to both index.md and README.md. Also added the gif to images/readme.